packages/react-instantsearch-dom-maps/src/__tests__/__snapshots__/GeoSearch.js.snap

```diff
@@ -1,17 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`GeoSearch Connector expect to render 1`] = `
-<AlgoliaGeoSearch(Connector)
+<Component
```

Comment:
This change is because we shallow render, a connector implies 2 wrapper components instead of one, the outer one is the context consumer, which doesn't have a display name; so this diff is expected. Once we move to hooks ☯️ we can cut down on that one layer of functions